README.md

```diff
@@ -64,3 +64,12 @@ Some aspects of the development will necessitate special care, they should be po
 ## Third Section: Future possibilities
 
 This last section talks about what has been thought of related to this issue, but has been decided not to be done now, and what it means regarding the feature at hand.
+
+## Review process
+
+The specification review is a two steps process:
+
+- 1) When the specification PR is opened (as draft), only ideas are discussed, until an agreement is found on the substance.
+- 2) When an agreement on the substance has been found, the PR is opened for review, and change requests to the shape are accepted.
```

Comment:
```suggestion
- 2) When an agreement on the substance has been found, the PR is opened for review, and change requests to the shape are accepted.
- 3) If any point is contested, the debate should not be about PR. It is therefore the duty of the organizer to call a meeting to discuss it. He will be responsible for writing in response to the thread the decisions taken. 
```

Reply:
I agree with @bidoubiwa, this point is general